src/main/kotlin/model/BackJack.kt

```diff
@@ -0,0 +1,55 @@
+package model
+
+import view.InputView
+import view.OutputView
+
+class BackJack() {
```

Comment:
```suggestion
class BlackJack() {
```

Maybe you missed an `l` here 👀